statusMapping.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+module.exports.STATUS_MAPPING = {
+  mapped: 1,
+  valid: 2,
+  disallowed: 3,
+  disallowed_STD3_valid: 4, // eslint-disable-line camelcase
```

Comment:
Could you use camel case here? `disallowedSTD3Valid`  Here and below.

Reply:
Actually I am using the kebab case version in generateTableMapping to map directly from the unicode [IdnaMappingTable](https://unicode.org/Public/idna/12.1.0/IdnaMappingTable.txt) status to single digits.

I could switch to camelcase and add a second mapping table, but I still would need to have those statuses.

Reply:
Ah got it. Thanks for clarifying!